proof_of_concept/definitions/assets.py

```diff
@@ -53,16 +54,25 @@ def __init__(self, id: Union[str, Identifier], data: Any,
         """
         if not isinstance(id, Identifier):
             id = Identifier(id)
+        if kind not in ('data', 'compute'):
```

Comment:
You could extract a constant `ASSET_KINDS`

Reply:
I hate you ;-)

Reply:
Haha, sorry :)